### PR TITLE
update to rc1.post1

### PR DESCRIPTION
**Pull request by geospatial-jeff**

### CODE REVIEW

File: .github/workflows/cicd.yml [hunk 1]
```diff
@@ -22,4 +22,4 @@ jobs:
         python-version: 3.8
     - name: Run tests
       run: |
-        pip install . && python setup.py pytest
\ No newline at end of file
+        pip install numpy && pip install . && python setup.py pytest
```
Comment (vincentsarago) [inline .github/workflows/cicd.yml:25]: I think you should use tox ;-)
https://github.com/cogeotiff/rio-tiler/blob/master/tox.ini#L8-L9

File: aiocogeo_tiler/tiler.py [hunk 2]
```diff
@@ -48,27 +49,16 @@ class COGInfo:
...
-            yield i
-
     @classmethod
     def from_array(cls, array: Union[np.ndarray, np.ma.masked_array]) -> "TileResponse":
         """create from numpy array"""
         if isinstance(array, np.ma.masked_array):
-            return cls(array.data, array.mask)
-        return cls(array)
+            return cls(data=array.data, mask=(array.mask * 255).astype("uint8"))
+        mask = np.zeros((array.shape[0], array.shape[1]), dtype="uint8") + 255
```
Comment (vincentsarago) [inline aiocogeo_tiler/tiler.py:60]: with rio-tiler==2.0.0rc2 I thin this can go

Comment (geospatial-jeff) [reply aiocogeo_tiler/tiler.py:60]: yeah :)

File: .github/workflows/cicd.yml (hunk 1 above)
Comment (geospatial-jeff) [reply .github/workflows/cicd.yml:25]: probably lol, a little bit of reinventing the wheel going on here

**Pull request merged**